{{cookiecutter.project_slug}}/config/settings/production.py

```diff
@@ -88,6 +88,8 @@
     "CacheControl": f"max-age={_AWS_EXPIRY}, s-maxage={_AWS_EXPIRY}, must-revalidate"
 }
 # https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html#settings
+AWS_S3_MAX_MEMORY_SIZE = env.int("AWS_S3_MAX_MEMORY_SIZE", default=100_000_000)  # 100MB
```

Comment:
To follow our standard, it probably needs to change to `DJANGO_AWS_S3_MAX_MEMORY_SIZE` and also update the settings table in the docs: https://github.com/cookiecutter/cookiecutter-django/blob/master/docs/settings.rst

```suggestion
AWS_S3_MAX_MEMORY_SIZE = env.int("DJANGO_AWS_S3_MAX_MEMORY_SIZE", default=100_000_000)  # 100MB
```